### PR TITLE
replaces the 88mod with a different gun

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
its literally just a sprite,typepath replacement
comments out the 88mod stock as nothing else uses it.

## Why It's Good For The Game
looks more in style
new
image
old
image

## Changelog
:cl:
add: Added new m18 sprite to replace 88mod sprite/lore thing
del: Removed old 88mod sprite, git will remember it, nobody else will. 88Mod stock commented out.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -61. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by XSlayer300** [comment]
Uhhh, where is `items_righthand_1.dmi`? You did not include it on the fiiles.

**Review by Hughgent — APPROVED** [review]

File: code/modules/projectiles/gun_attachables.dm [hunk 1]
```diff
@@ -931,7 +931,8 @@ Defined in conflicts.dm of the #defines folder.
 	scatter_mod = 0
 	movement_acc_penalty_mod = 0
 
-/obj/item/attachable/stock/vp70
+/*
+/obj/item/attachable/stock/vp70 //The gun that uses this no longer exists. So its commented out
 	name = "88 Mod 4 stock and holster"
 	desc = "A rare holster-stock distributed in small numbers to TGMC forces. Compatible with the 88 Mod 4, this stock reduces recoil and improves accuracy, but at a reduction to handling and agility. Seemingly a bit more effective in a brawl."
```
Comment (TiviPlus) [inline code/modules/projectiles/gun_attachables.dm:937]: do not comment out code just remove it

**Comment by Pariah919** [comment]
i'll reopen this another day, another time

**Pull request closed without merging**